R/delta.R

```diff
@@ -1,14 +1,198 @@
 
 
-#' Title
+#' Create a delta data.frame template
 #'
 #' @description
-#' TODO
+#' Creates a data.frame in the format required by [analyse()] for the use
+#' of applying delta to perform a tipping point analysis.
```

Comment:
As said before, I would not say "tipping point analysis". I would say "Creates a data.frame in the format required by [analyse()] for the use of applying a delta-adjustment strategy".

Reply:
done